.c8rc

```diff
@@ -0,0 +1,18 @@
+{
+    "reporter": ["text", "html", "lcov"],
```

Comment:
Even though vscode-extension-tester has a built-in default c8 options, I needed to prepare this for

- Add "lcov" to "reporter"
- Exclude syntaxHighlighter from coverage calculation.  It is proviided as a bundled js, coverage collection does not work well.